src/main/java/org/projectrainbow/RainbowLauncher.java

```diff
@@ -17,7 +17,7 @@
 
     public static void main(String[] args) throws Exception {
         File rainbowJar = new File("Rainbow.jar");
-        downloadFile("https://ci.codecrafter47.de/job/Rainbow/lastSuccessfulBuild/artifact/rainbow/target/Rainbow.jar", rainbowJar);
+        downloadFile("http://ci.codecrafter47.de/job/Rainbow/lastSuccessfulBuild/artifact/rainbow/target/Rainbow.jar", rainbowJar);
```

Comment:
I'd prefer using the ssl encrypted connection. The latest Java 8 release will accept the certificate.